packages/toast/toast.js

```diff
@@ -1,40 +1,31 @@
 import { css, html } from 'lit';
 
+import { classNames } from '@chbphone55/classnames';
 import { i18n } from '@lingui/core';
 import { toast as ccToast } from '@warp-ds/css/component-classes';
 import WarpElement from '@warp-ds/elements-core';
 import { expand, collapse } from 'element-collapse';
-import { classMap } from 'lit/directives/class-map.js';
 import { when } from 'lit/directives/when.js';
 import '@warp-ds/icons/elements/warning-16';
 import '@warp-ds/icons/elements/error-16';
 import '@warp-ds/icons/elements/success-16';
 import '@warp-ds/icons/elements/close-16';
 
 import { activateI18n } from '../i18n';
+import { kebabCaseAttributes } from '../utils';
 
 import { messages as daMessages } from './locales/da/messages.mjs';
 import { messages as enMessages } from './locales/en/messages.mjs';
 import { messages as fiMessages } from './locales/fi/messages.mjs';
 import { messages as nbMessages } from './locales/nb/messages.mjs';
 
-const classes = (definition) => {
-  const defn = {};
-  for (const [key, value] of Object.entries(definition)) {
-    for (const className of key.split(' ')) {
-      defn[className] = value;
-    }
-  }
-  return classMap(defn);
-};
-
 const toastType = {
   success: 'success',
   error: 'error',
   warning: 'warning',
 };
 
-export class WarpToast extends WarpElement {
+export class WarpToast extends kebabCaseAttributes(WarpElement) {
```

Comment:
same here

Reply:
Updated in this commit: [4cdf72ba](https://github.com/warp-ds/elements/pull/177/commits/4cdf72ba11a40ec6e98e8c18f1744141a4c4159d)